index.html

```diff
@@ -173,6 +173,44 @@ <h2>Calls for implementation and feedback</h2>
+
+<p>For features advancing from Stage 1 to Stage 2, a delegate may pose a constraint as necessary for advancement to Stage 3. In this situation, the delegate should expect to work with the champion and other delegates during Stage 2 to incorporate their constraint into the solution. In general, this is a good time to pose new constraints.
+
+<p>For features advancing to Stage 3, a delegate may ask that the champion work with them to incorporate a new constraint as a condition of advancement to Stage 3. In particular, given that consensus on Stage 3 means "the solution is complete", i.e., all open design issues havee beeen resolved including anticipated implementation and ecosystem compatibility issues, all TC39 participants should validate the design of proposals they care about before granting Stage 3 consensus.
+
+<p>Delegates should openly give feedback on proposals, especially when they have significant concerns, and especially for a proposal for stage advancement where the concern is relevant to the stage. Even if a Delegate does not intend to withhold consensus they should raise their concerns for general discussion.
+
+<p>A delegate may also request additional time to consider the proposal.
+
+<p>Conditional advancement is time-limited, giving the person raising the concern time to discuss with the champions and authors about their concerns. If a proposal has a conditional advancement, an issue must be opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.
+
+<p>The committee must record a good description of why a proposal did not advance, both in the meeting notes and within an issue in the proposal's tracker, but not limited to those. So that in future meetings and future proposals we can learn from failed proposals, and have a good record of why they failed.
+
+<p>In order to ensure that the committee is able to do it's work effectively, there are restrictions to witholding consensus.
+
+<p>The ability to withhold consensus for reaching stage 1, 2 and 3 is not limited but constraints introduced should be weighed according to their stage. Proposals which have fulfilled the acceptance criteria may be blocked from advancement for any well stated reason.
+
+<p>The ability to withold consesus for reaching stage 4 is limited. Proposals which have fulfilled the acceptance criteria may not be withhold advancement unless the constraint is related to implementation experience or identifies an issue or information which has not previously been discussed by the committee. Withholding consensus for stage 4 advancement is limited, as during stage 3 implementers  is to allow implementers to invest in implementations, and maintain the significance of stage 3 in the process.
+
+<h2>Reverting to Earlier Stages</h2>
```

Comment:
What we're talking about here applies for withdrawing proposals as well, so we might as well document that.

```suggestion
<h2>Withdrawing Proposals and Reverting to Earlier Stages</h2>
```